File: api-tester/src/main/java/com/revenuecat/apitester/java/BillingFeatureAPI.java
package com.revenuecat.apitester.java;

import com.revenuecat.purchases.models.BillingFeature;

@SuppressWarnings({"unused", "SpellCheckingInspection"})
final class BillingFeatureAPI {
    static void check(final BillingFeature feature) {
        switch (feature) {
            case SUBSCRIPTIONS:
            case SUBSCRIPTIONS_UPDATE:
            case PRICE_CHANGE_CONFIRMATION:
        }
    }
}

File: api-tester/src/main/java/com/revenuecat/apitester/kotlin/BillingFeatureAPI.kt
package com.revenuecat.apitester.kotlin

import com.revenuecat.purchases.models.BillingFeature

@Suppress("unused", "UNUSED_VARIABLE")
private class BillingFeatureAPI {
    fun check(billingFeature: BillingFeature) {
        when (billingFeature) {
            BillingFeature.PRICE_CHANGE_CONFIRMATION,
            BillingFeature.SUBSCRIPTIONS,
            BillingFeature.SUBSCRIPTIONS_UPDATE,
            -> {
            }
        }.exhaustive
    }
}
